common/src/main/resources/db/changelog/v001/ab2d-934-allow-for-zip-downloads.sql

```diff
@@ -0,0 +1,7 @@
+--liquibase formatted sql
+--  -------------------------------------------------------------------------------------------------------------------
+
+--changeset lsharshar:ab2d-934-allow-for-zip-downloads failOnError:true
+ALTER TABLE job ADD COLUMN output_format VARCHAR(255);
+
+--rollback  ALTER TABLE output_format DROP COLUMN IF EXISTS mbi;
```

Comment:
The `rollback` statement needs to be updated to drop column `output_format` from `job` table.

Reply:
Whoops, copy and paste.